.github/.yamllint

```diff
@@ -0,0 +1,13 @@
+---
+extends: default
```

Comment:
Your config doesn't seem to load based on my testing.

https://github.com/Mab879/content/actions/runs/15718684713/job/44294749074?pr=43

Reply:
Thanks @Mab879 . It should be working now by my tests [in my fork.](https://github.com/marcusburghardt/content/actions/runs/15719053628/job/44295956081?pr=8)

I only moved the config file for the root and removed the `--no-warnings` option since warnings don't fail but are good to be shown. Since the changes were minimal, I amended and force pushed to keep the master cleaner. : )